example/spec/loadTest/queueGranulePassThroughLoadTest.js

```diff
@@ -0,0 +1,253 @@
+  } else {
+    console.log(redConsoleLog(), `\n___ Execution of Workflow ${nthWorkflow} Failed ___\n ${JSON.stringify(workflowExecution)}\n`);
+  }
+
+  const lambdaOutput = await new LambdaStep().getStepOutput(
+    workflowExecution.executionArn,
+    'QueueGranules'
+  );
+  queueGranulesChildExecutionArns.push(lambdaOutput.payload);
+
+  if (lambdaOutput.payload.running.length === granuleCountPerWorkflow) {
+    colorConsoleLog = greenConsoleLog();
+  } else {
+    colorConsoleLog = redConsoleLog();
+    console.log(colorConsoleLog, '\n___ lambdaOutputPayload ExecutionArns ___');
+    lambdaOutput.payload.running.map((runningArn) =>
+      console.log(`${JSON.stringify(runningArn)}`));
+  }
+  console.log(colorConsoleLog,
+    `\n___ Lambda Output of Workflow ${nthWorkflow} has ${lambdaOutput.payload.running.length}/${granuleCountPerWorkflow} of expected arns ___`);
+
+  const completedGranules = [];
+  const incompletedGranules = [];
+
+  const expectedValues = ['completed'];
+
+  await Promise.all(
+    inputPayload.granules.map(async (granule) => {
+      try {
+        const record = await waitForApiStatus(
+          getGranule,
+          {
+            prefix: config.stackName,
+            granuleId: granule.granuleId,
+          },
+          expectedValues
+        );
+        completedGranules.push(record);
+        totalGranulesCompleted.push(record);
+      } catch (error) {
+        incompletedGranules.push({ granuleId: granule.granuleId, err: error });
+      }
+    })
+  );
+
+  console.log(greenConsoleLog(),
+    `\n___ ${completedGranules.length}/${inputPayload.granules.length} Granules completed by workflow ${nthWorkflow} ___`);
+
+  if (completedGranules.length !== inputPayload.granules.length) {
+    console.log(redConsoleLog(),
+      `\n___ ${incompletedGranules.length}/${inputPayload.granules.length} Granules incompleted by workflow ${nthWorkflow} ___`);
+
+    incompletedGranules.map((granuleError) =>
+      console.log(redConsoleLog(),
+        `\n${JSON.stringify(granuleError.granuleId)} - ${granuleError.err}`));
+  }
+};
+
+describe('The Granule Ingest Load Test ', () => {
+  beforeAll(async () => {
+    if (!Number.isFinite(Number(granuleCountPerWorkflow)) || granuleCountPerWorkflow > granuleThresholdPerWorkflow) {
+      beforeAllFailed = `===== beforeAll() - Invalid input for number of granules per Workflow detected - ${granuleCountPerWorkflow} =====`;
+      throw new Error(beforeAllFailed);
+    } else {
+      try {
+        config = await loadConfig();
+
+        process.env.GranulesTable = `${config.stackName}-GranulesTable`;
+        process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
+        process.env.CollectionsTable = `${config.stackName}-CollectionsTable`;
+
+        await pMap(
+          new Array(totalWorkflowCount).fill().map((_, index) => index + 1),
+          async (index) => {
+            await batchGranulesProcessing(index);
+          },
+          { concurrency: 8 }
```

Comment:
This should probably be a configurable variable, or possibly set to totalWorkflowCount?         Some of that depends on the outcome of the workflow/parallelism discussion.